helm/kubernetes-gpu-app/templates/device-plugin-ds.yaml

```diff
@@ -70,7 +37,7 @@ spec:
         hostPath:
           path: /dev
       containers:
-      - image: "k8s.gcr.io/nvidia-gpu-device-plugin@sha256:d18b678437fedc4ec4211c20b3e5469a137a44f989da43dc275e4f2678170db4"
+      - image: quay.io/giantswarm/nvidia-gpu-device-plugin:1.0.0
```

Comment:
I think the tag at least should be in values file

Reply:
hmm I forget to include this file, good catch